### DISCUSSION TITLE
TUTORIAS EDA 2

### DISCUSSION THREAD

**Discussion by Dievex** [mmasias/23-24-eda2#40] [General]
Viendo la situación tan ambigua que hubo en clase con respecto al cambio de horario de la asignatura, he hablado con @mmasias acerca de si fuera posible para aquella gente que tenga dificultades con el horario hacer tutorías para que así todo el mundo pueda ponerse al día con la asignatura. Hemos acordado esta tutoría hacerla los lunes de 15:00 a 16:00. 

Un saludo,
Diego García Niño

**Comment by mmasias** [comment]
En el aula 0.2, a las 15:00h del lunes
